app/views/claims/information_provided.html.erb

```diff
@@ -35,8 +42,26 @@
         </span>
       </summary>
       <div class="govuk-details__text">
-        We send HMRC your full name, date of birth, payment amount and gender details to ensure your
-        tax contributions are met.
+        <p class="govuk-body">
+          We send Cantium and HMRC your:
```

Comment:
Now that this includes information that we send Cantium do we need to update the summary? https://github.com/DFE-Digital/dfe-teachers-payment-service/blob/bf59e70dbf6526ccbcee1d93835a24e78613452f/app/views/claims/information_provided.html.erb#L41